wrappers/services.go

```diff
@@ -397,6 +510,10 @@ func ServicesEnableState(s *snap.Info, inter interacter) (map[string]bool, error
 		if !app.IsService() {
 			continue
 		}
+		// FIXME: handle user daemons
+		if app.DaemonScope != snap.SystemDaemon {
+			continue
+		}
```

Comment:
Is this for a followup?

Reply:
Yes.  It isn't clear how best to handle this given that individual users could enable/disable units independently.